composer.json

```diff
@@ -34,6 +33,7 @@
         "nelmio/api-doc-bundle": "~2.4"
     },
     "require-dev": {
+        "sonata-project/google-authenticator": "~1.0",
```

Comment:
As mentioned in https://github.com/sonata-project/SonataMediaBundle/pull/753 this should be added to **require-dev** for travis.